code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -491,3 +491,131 @@
 	if(prob(15))
 		M.adjustToxLoss(2, 0)
 	..()
+
+/datum/reagent/drug/maint
+	name = "Maintanance Drugs"
```

Comment:
botany can make this chem with strange seeds

Reply:
how do i make it so that it doesnt happen?

Reply:
make it a blacklisted chem, and make its subtypes not be blacklisted

Reply:
can_synth to false but you'll have to override the subtypes.

Reply:
`Maintenance`